bedrock/settings/__init__.py

```diff
@@ -19,53 +20,86 @@
 
 ROOT_URLCONF = "bedrock.urls"
 
-# CSP settings for MOZORG, expanded upon later:
+# CSP settings, expanded upon later:
+# NOTE: We are providing all settings to django-csp as sets, not lists.
+# - This is for de-duping, and because django-csp will convert them to `sorted` lists for us.
+
+# NOTE: For any URLs that contain a path, not just the origin, trailing slashes are important.
+# - if no path is provided, all resources are allowed from the origin.
+# - if path is provided with no trailing slash: an exact-match is required.
+#   - e.g. `https://example.com/api` will only match `https://example.com/api`
+# - if path is provided with trailing slash: the path is a prefix-match.
+#   - e.g. `https://example.com/api/` will match anything that starts with `https://example.com/api/`
+
 _csp_default_src = {
+    # NOTE: Keep `default-src` minimal. Best to set resources in the specific directives.
+    csp.constants.SELF,
+}
+_csp_connect_src = {
+    csp.constants.SELF,
+    # NOTE: Check if these need to be in the `_csp_form_action` list as well since we often
+    # progressively enhance forms by using Javascript.
+    "o1069899.ingest.sentry.io",
+    "o1069899.sentry.io",
+    "region1.google-analytics.com",
+    "www.google-analytics.com",
+    "www.googletagmanager.com",
+    # This is for glean pings.
+    "www.mozilla.org/submit/bedrock/events/",
```

Comment:
Looks like we may also need to cover `https://www.mozilla.org/submit/bedrock/deletion-request/1/e4f5408d-f83d-4709-995a-7ab0955611b5`, which is a different event type when someone rejects analytics and we send a request to delete any previous pings.

Reply:
Maybe we could simplify this to just `www.mozilla.org/submit/bedrock/` ?

Reply:
Great, I'll make that change.